app/helpers/fhir_helper.rb

```diff
@@ -158,39 +173,54 @@ def patient_from_fhir(patient, default_jurisdiction_id)
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2106-3', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'White')
       ) : nil,
-      black_or_african_american ? FHIR::Extension.new(
+      races[:black_or_african_american] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2054-5', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Black or African American')
       ) : nil,
-      american_indian_or_alaska_native ? FHIR::Extension.new(
+      races[:american_indian_or_alaska_native] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '1002-5', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'American Indian or Alaska Native')
       ) : nil,
-      asian ? FHIR::Extension.new(
+      races[:asian] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2028-9', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Asian')
       ) : nil,
-      native_hawaiian_or_other_pacific_islander ? FHIR::Extension.new(
+      races[:native_hawaiian_or_other_pacific_islander] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2076-8', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Native Hawaiian or Other Pacific Islander')
       ) : nil,
+      races[:race_unknown] ? FHIR::Extension.new(
+        url: 'ombCategory',
+        valueCoding: FHIR::Coding.new(code: 'UNK', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Unknown')
+      ) : nil,
+      races[:race_other] ? FHIR::Extension.new(
+        url: 'ombCategory',
+        valueCoding: FHIR::Coding.new(code: 'OTH', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Other')
+      ) : nil,
+      races[:race_refused_to_answer] ? FHIR::Extension.new(
+        url: 'ombCategory',
+        valueCoding: FHIR::Coding.new(code: 'ASKU', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Refused to Answer')
+      ) : nil,
       FHIR::Extension.new(
         url: 'text',
-        valueString: [white ? 'White' : nil,
-                      black_or_african_american ? 'Black or African American' : nil,
-                      american_indian_or_alaska_native ? 'American Indian or Alaska Native' : nil,
-                      asian ? 'Asian' : nil,
-                      native_hawaiian_or_other_pacific_islander ? 'Native Hawaiian or Other Pacific Islander' : nil].reject(&:nil?).join(', ')
+        valueString: [races[:white] ? 'White' : nil,
+                      races[:black_or_african_american] ? 'Black or African American' : nil,
+                      races[:american_indian_or_alaska_native] ? 'American Indian or Alaska Native' : nil,
+                      races[:asian] ? 'Asian' : nil,
+                      races[:native_hawaiian_or_other_pacific_islander] ? 'Native Hawaiian or Other Pacific Islander' : nil,
+                      races[:race_unknown] ? 'Unknown' : nil,
```

Comment:
The api validation and this task were being done in parallel so I didn't add it at the time. I can definitely put it in now, but I think it probably would make more sense to error if an exclusive and another field are both set to true, rather than default to picking one.